policyengine_us/parameters/gov/states/ct/tax/income/subtractions/social_security/magi_excess.yaml

```diff
@@ -0,0 +1,12 @@
+description: Connecticut calculates this fraction of the modified adjusted gross income excess over base amount, defined in Section 86(b)(1) of the Internal Revenue Code, under the social security benefit adjustment.
```

Comment:
I meant clarify as in describe what the policy relevance of this parameter is. Please replace the IRC section with a description of how this enters into the SS subtraction. ChatGPT may be able to help if you pass it the legal code and/or tax form and ask it to describe the purpose of the 25%.